src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -45,26 +54,49 @@ public static TryCatchStatement composeRpcMethodSampleCode(
       return composeLroUnaryRpcMethodSampleCode(method, arguments, clientType);
     }
     // Pure Unary RPC method.
-    return composeUnaryRpcMethodSampleCode(method, arguments, clientType);
+    return composeUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
   }
 
   private static TryCatchStatement composeUnaryRpcMethodSampleCode(
-      Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary rpc method.
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add unit tests.
     VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    // Assign each method arguments with default value.
+    List<Expr> bodyExpr =
+        arguments.stream()
+            .map(methodArg -> assignMethodArgumentWithDefaultValue(methodArg, resourceNames))
```

Comment:
Note that if `assignMethodArgumentWithDefaultValue` will only ever be called from a stream, it may be more efficient to make it take `arguments` instead.

Reply:
sg